src/main/groovy/com/bmuschko/gradle/docker/DockerJavaApplicationPlugin.groovy

```diff
@@ -18,6 +18,7 @@ package com.bmuschko.gradle.docker
 import com.bmuschko.gradle.docker.tasks.image.DockerBuildImage
 import com.bmuschko.gradle.docker.tasks.image.DockerPushImage
 import com.bmuschko.gradle.docker.tasks.image.Dockerfile
+import groovy.transform.PackageScope
```

Comment:
Good catch. I would expect that there would be some sort of style linter that would have caught that for me like the [spotless](https://github.com/diffplug/spotless/tree/master/plugin-gradle) plugin. I'll go and remove it.